src/routing.test.js

```diff
@@ -0,0 +1,59 @@
+import { parsePath } from './routing'
+import { APP_MODE_START, APP_MODE_ORG } from './symbols'
+
+const ADDRESS = '0xc41e4c10b37d3397a99d4a90e7d85508a69a5c4c'
+
+function locator(path, data) {
+  return {
+    path,
+    pathname: path,
```

Comment:
Shouldn't this be `path.split('/')`?

Reply:
No, `pathname` is `path` without the search part of the URL: https://github.com/aragon/aragon/blob/app-paths/src/routing.js#L44-L47

I could add support for a complete path in this utility function, but we would enter in the territory of what we are trying to test :D

Reply:
I made it a bit less weird − hopefully :sweat_smile: https://github.com/aragon/aragon/pull/846/commits/82ab8f70b77df3db950a50d9c89430e4ef6339b4